src/components/button/Button.tsx

```diff
@@ -83,7 +116,8 @@ class Button extends React.Component<ButtonProps> {
                 type={type}
                 {...buttonProps}
             >
-                <span className="btn-content">{children}</span>
+                {textContent}
```

Comment:
Should we continue to move toward inlining our renders and/or moving more complex inner render logic to separate components?

Reply:
It actually looks OK inlined, I'll do that